docs/tutorials/elasticdl_deepctr.md

```diff
@@ -0,0 +1,279 @@
+# ElasticDL：Distributed Train the DeepCTR Model on Kubernetes
+
+This document shows how to use ElasticDL to train the premade Keras Model
+on Kubernetes. In the document,  we use the model in DeepCTR which is a
+popular package in CTR estimation and the dataset provided by Cretio from
+Kaggle Advertising Challenge contest. There are 13 columns of integer
+features I1-I13 and 26 columns of categorical features C1-C26 in the dataset.
+
+## 1. Prerequisites
+
+- Install Minikube, preferably >= v1.11.0, following the installation guide.
+Minikube runs a single-node Kubernetes cluster in a virtual machine on your
+personal computer.
+- Install Docker CE, preferably >= 18.x, following the guide for building
+Docker images containing user-defined models and the ElasticDL framework.
+- Install Python, preferably >= 3.6, because the ElasticDL command-line
+tool is in Python.
+- Install [pyrecordio](https://pypi.org/project/pyrecordio/),
+[deepctr](https://pypi.org/project/deepctr/) and TensorFlow >= 2.0,
+[elasticdl_client](https://pypi.org/project/elasticdl-client/).
+
+## 2. Convert Data into RecordIO
+
+ElasticDL requires that the sample data can be queried by an integer
+index, so users need to convert the training data in [criteo_sample.txt](https://raw.githubusercontent.com/shenweichen/DeepCTR/master/examples/criteo_sample.txt)
+into RecordIO files.
+
+```python
+import argparse
+import os
+import pathlib
+import sys
+
+import recordio
+import tensorflow as tf
+
+COLUMNS = (
+    ["label"]
+    + ["I" + str(i) for i in range(1, 14)]
```

Comment:
["I" + str(i) for i in range(1, 14)] => [("I" + str(i)) for i in range(1, 14)]
for readability.